test/inductor/test_aot_inductor.py

```diff
@@ -973,6 +973,7 @@ def forward(self, x):
         example_inputs = (torch.randn(8, 4, 4, device=self.device),)
         self.check_model(Model(), example_inputs)
 
+    @unittest.skipIf(IS_FBCODE, "Not runnable in fbcode")
```

Comment:
Wondering why this test is not runnable in fbcode? Does the change of using ProxyExecutor as fallback by default affect this test? Thanks.

Reply:
In OSS ABI mode, this test is failing due to aten.index.Tensor missing c-shim, this is why we have it as xfail in the first place. 
In fbcode ABI mode, this is failing due to extern_kernel_serializer not populated in AOTInductorModelRunner. I tried to add it, but this introduces dependency of fbcode module ‘deeplearning.aot_inductor.extern_node_thrift_serializer’ into an oss test file, which buck build failed. 

As a result, I added the same test case to `test_custom_op.py`, as you can see in the fbcode diff.